lib/ecto/multi.ex

```diff
@@ -274,17 +274,17 @@ defmodule Ecto.Multi do
   The function should return either `{:ok, value}` or `{:error, value}`, and
   receives changes so far as an argument.
   """
-  @spec run(t, name, run) :: t
-  def run(multi, name, run) when is_function(run, 1) do
-    add_operation(multi, name, {:run, run})
+  @spec run(t, name, fun) :: t
```

Comment:
@lukerollans oh I see, the spec does not match the function arguments.

We should change `run` in the spec to `(t -> {:ok | :error, any})`. Could you fix that?